docs/pages/reference/terraform/index.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: "Teleport Terraform Provider"
+description: |-
+  
+---
+
+The Teleport Terraform provider allows Terraform users to configure Teleport
+from Terraform.
+
+This section is the Teleport Terraform Provider reference.
+It lists all the supported resources and their fields.
+
+<Admonition type="tip">
+To get started with the Terraform provider, you must start with the installation guide.
```

Comment:
Is this referring to the main Teleport installation guide or [Set up the Teleport Terraform Provider](https://goteleport.com/docs/management/dynamic-resources/terraform-provider/)?